docs/user-guide/images.md

```diff
@@ -100,6 +101,23 @@ Troubleshooting:
   - `plugins.go:56] Registering credential provider: aws-ecr-key`
   - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
 
+### Using Azure Container Registry (ACR)
+When using Azure container registry you can authenticate using either an admin user or a service principal.
+In either case, authentication is done via standard Docker authentication.  These instructions assume the
+python based [az](https://github.com/azure/azure-cli) tooling.
+`
+```console
+# To generate new credentials
+az acr credential renew -n ${CLUSTER_NAME} -g ${CLUSTER_RESOURCE_GROUP}
+
+# To show existing credentials
```

Comment:
Correct. This is rendering as an H1 in mid-document; please check your formatting.